wordpress-mobile/WordPress-Android
# Add support for taking photos for site icon/featured image selection

This PR adds support for camera for site icon and featured image selection. I didn't add the option to take videos cause I couldn't think of a use case where it would be used. It was partially implemented in the previous picker but never fully functional. We can add it if we ever want to use it.

To test:
- Turn on the consolidated media picker flag
- Click on site icon
- In the media picker click on the FAB 
- Take a photo
- Editor is opened
- Confirm edited photo
- Notice the photo gets set as the site icon

To test:
- Turn on the consolidated media picker flag
- Go to Me screen
- Click to change avatar
- In the media picker click on the FAB 
- Take a photo
- Editor is opened
- Confirm edited photo
- Notice the photo gets set as the site icon

PR submission checklist:
